### DISCUSSION TITLE
[Projects Beta] Is adding multiple workflows being worked on?

### DISCUSSION THREAD

**Discussion by sakibarifin** [community/community#32905] [Projects and Issues]
In the Workflows menu, we can only add one action.

![image](https://user-images.githubusercontent.com/33556544/189908129-e191114c-146c-40f9-b9c1-37d2a8d2ca35.png)

It would be great if we can add multiple statuses like priority, size etc by default. Is it being worked on?

And any possibility to get a csv or excel export option? It would be even better if projects supported connections between google sheets and projects with the api for personal projects not organization projects.

Trello has the option to move cards from one board to another. If we could transfer a card / row from one project, it would help with transition from Trello.

**Comment by Cherlepops** [comment]
+1 
Also It would be great if we could set an action for each type of items, because you might not want the same action being done depending if it's an issue or a pull request

**Reply by MattMcAdams** [reply]
Yeah I'm wanting to have new issues go into a "Triage" status and new pull requests to go into a "Needs Review" status, so it would be nice to have different workflows based on item type.
